spec/bourbon/css3/font_face_spec_2.rb

```diff
@@ -0,0 +1,19 @@
+require "spec_helper"
+
+describe "font-face" do
+  before(:all) do
+    ParserSupport.parse_file("css3/font-face-2")
+  end
+
+  context "called with defaults" do
+    it "outputs defaults" do
+      ruleset = 'font-family: "roboto"; ' +
+                'font-style: "italic"; ' +
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.